include/cx.h

```diff
@@ -182,13 +183,21 @@ class LocationInfoAnswer : public Diameter::Message
                      const uint32_t& vendor_id,
                      const int32_t& experimental_result_code,
                      const std::string& scscf,
-                     const ServerCapabilities& capabs);
+                     const ServerCapabilities& capabs,
+                     const std::string& wildcarded_pub_id = "");
   inline LocationInfoAnswer(Diameter::Message& msg) : Diameter::Message(msg) {};
 
   inline bool server_name(std::string& str) const
   {
     return get_str_from_avp(((Cx::Dictionary*)dict())->SERVER_NAME, str);
   }
+
+  inline bool wildcarded_pub_id(std::string& str) const
```

Comment:
Might as well have this be wildcarded_public_identity as the name (same comment throughout)